other/field-types/time.md

```diff
@@ -15,3 +17,8 @@ The default "local" time format, displayed to the user when editing, is American
 | name | `string` | | Sets the name of the field in the database |
 | label | `string` | | Sets the label of the field that the user sees |
 | userTimeFormat |  |  | Allows configuration of the time format |
+| def | | | Sets the default time that is displayed |
+
+{% hint style='info' %}
+If you do not set `def: null` or `required: true`, the time defaults to the current time.
```

Comment:
I just noticed that `required` isn't in these tables. So some universal properties are in the tables but others aren't. Should we have a link on each noting that other universal ones are listed on the other page?

Reply:
I think all of the universal properties should be listed in all of the tables on the type specific pages. You've come to a page just for this field type, there's ample room to show you everything you're looking for. We could move them to the end or something if desired.

Reply:
I was going back and forth on that, and I'll update it so that everything has the full list of properties.

Reply:
Tables are now complete.